web/src/components/DLP/OverviewTab.vue

```diff
@@ -165,7 +165,7 @@
               v-else-if="!assetSummary || !Object.keys(assetSummary).length"
               class="font-italic font-weight-bold"
             >
-              This Dandiset does not contain any assets.
+              This Dandiset does not contain any valid assets.
```

Comment:
I know this is technically the most concise way to phrase this, but I wonder if people will take both wordings to mean the same thing. They might say "hold on, I just uploaded a bunch of assets that I _know_ are valid", even though "valid" might mean different things to us vs them. Maybe we should reword it to something like:

`This dandiset does not contain any valid assets. Please check the validation errors to address any issue with asset metadata.`, or something like that :man_shrugging: .

Reply:
I added a message about the validation errors that only appears when there are asset validation errors - https://github.com/dandi/dandi-archive/pull/2122/commits/9fcf1dd8053a0a96ec4dd2c06f980b30ce8bb01b. What do you think?

Reply:
I think the logic in that commit is inverted. This can be seen in [this](https://deploy-preview-2122--gui-staging-dandiarchive-org.netlify.app/dandiset/101233) example. It should show the extended message, but just shows the short one. While the [other](https://deploy-preview-2122--gui-staging-dandiarchive-org.netlify.app/dandiset/215957) example shows the extended message, even though it has zero assets.

Reply:
Besides that I think the extended message is perfect.

Reply:
Oh I inverted the logic to test it, and forgot to change it back :man_facepalming: . Thanks for catching that, I fixed it